libosmscout-import/src/osmscoutimport/GenRelAreaDat.cpp

```diff
@@ -322,7 +322,7 @@ namespace osmscout {
       }
     }
 
-    parts=rings;
+    std::move(rings.begin(), rings.end(), std::back_inserter(parts));
```

Comment:
this changes the semantics, or not? Should not we call `parts.clear()` before adding entries? or possibly `assert(parts.empty())`

Reply:
I believe that it should look like this: https://github.com/Framstag/libosmscout/pull/1322